homeassistant/components/template/light.py

```diff
@@ -178,10 +229,14 @@ def name(self):
     @property
     def supported_features(self):
         """Flag supported features."""
+        supported_features = 0
         if self._level_script is not None:
-            return SUPPORT_BRIGHTNESS
-
-        return 0
+            supported_features |= SUPPORT_BRIGHTNESS
+        if self._temperature_script is not None:
```

Comment:
Interesting that this is based off the existence of a script, not on a value template ?

Reply:
Indeed, I just follow the existing solution. But IMO it seems logic: you need an action to be able to change the brightness/color/temperature.